src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -2332,6 +2332,7 @@ void MusicXMLParserPass2::measure(const String& partId, const Fraction time)
     InferredFingeringsList inferredFingerings; // Directions to be reinterpreted as Fingerings
     ArpeggioMap arpMap;
     DelayedArpMap delayedArps;
+    bool coda = false;
```

Comment:
It's not used in this function. What exactly is the purpose of this flag?

Reply:
This is a flag to show the presence of a coda in the measure.  More than one coda in a measure is usually a mistake, so after this is true we don't add any more codas for the remainder of the measure.  I've renamed it to `measureHasCoda` for clarity.